doc/documentation/src/tutorial_templates/tutorial_preconditioning.md.j2

```diff
@@ -10,7 +10,21 @@
 {% set input_file_solid_amg_content = load_input_file(tutorial_file_solid_amg) %}
 
 (preconditioningtutorial)=
-# Preconditioning Tutorial
+# Tutorial: Preconditioners for Iterative Solvers
+
+Preconditioners are a key tool to accelerate iterative solvers for large linear systems, improving convergence and robustness.
+From a theoretical perspective, they can be understood as transformations that reduce the condition number of the system matrix, thereby making iterative methods converge more rapidly.
+
+For a broader, theoretical introduction to iterative methods and preconditioning, see Saad's *Iterative Methods for Sparse Linear Systems* [^1].
+Multiple sources for a theoretical background on Multigrid Methods can be found, e.g., in Briggs et al., *A Multigrid Tutorial* [^2].
+For an introduction to the use of linear solvers in 4C, we refer to this {% raw %}{ref}`overview <solver-interfaces>`{% endraw %}.
+
+
+```{note}
+In practical applications, the primary performance metric for simulation engineers is usually computation time rather than iteration counts.
+However, in this tutorial we deliberately focus on **iteration counts**: the small problem sizes chosen here make timing results unreliable and not representative.
+For realistic, large-scale problems, timings are of course essential, but for illustrative purposes iteration counts are the more meaningful measure.
+```
```

Comment:
```suggestion
```{note}
In practical applications, the primary performance metric for simulation engineers is usually computation time rather than iteration counts.
However, in this tutorial we deliberately focus on **iteration counts**: the small problem sizes chosen here make timing results unreliable and not representative.
In contrast to timings, iteration counts are reproducible across problem sizes, hardware platforms and software configurations.
For realistic, large-scale problems, timings are of course essential, but for illustrative purposes iteration counts are the more meaningful measure.
```
```